specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2020-09-01",
+    "description": "To manage and control access to your resources, you can define customized policies and assign them at a scope."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/dataPolicyManifests/{policyMode}": {
+      "get": {
+        "tags": [
+          "DataPolicyManifests"
+        ],
+        "operationId": "DataPolicyManifests_GetByPolicyMode",
+        "summary": "Retrieves a data policy manifest.",
+        "description": "This operation retrieves the data policy manifest with the given policy mode.",
+        "x-ms-examples": {
+          "Retrieve a data policy manifest by policy mode": {
+            "$ref": "./examples/getDataPolicyManifest.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyMode",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy mode of the data policy manifest to get."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about a data policy manifest.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifest"
+            }
+          },
+          "404": {
+            "description": "Not found - the data policy manifest doesn't exist."
```

Comment:
I think you should add the schema section here too, as in default case

Reply:
If I add the schema the validation asks for a sample 404 response body, as I couldn't find any example for this I ended up removing the schema

Reply:
Declaring explicit 404 response has a side effect on the generated SDK. Instead of throwing an exception, it will generate a bool return false or something like that IIRC. Are you sure you actually want that? Most RPs do not declare explicit status codes.

---
In reply to: [513108761](https://github.com/Azure/azure-rest-api-specs/pull/11391#discussion_r513108761) [](ancestors = 513108761)

Reply:
What do you say @varunkumta? 
I think since not declaring the response explicitly still means that an exception will be returned, there shouldn't be a need for it, right?